### PR TITLE
OSDOCS 7476: Adding Build and DeploymentConfig to the list of optiona…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…l capabilities

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-7476

Link to docs preview:
https://63812--docspreview.netlify.app/openshift-enterprise/latest/installing/cluster-capabilities

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@ingvagabund Here's the PR for adding these APIs to the capabilities page. Can you PTAL?

Preview: https://63812--docspreview.netlify.app/openshift-enterprise/latest/installing/cluster-capabilities

File: modules/build-config-capability.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// *  installing/cluster-capabilities.adoc
+
+:_content-type: REFERENCE
+[id="build-config-capability_{context}"]
+= Build capability
```
Comment (ingvagabund) [inline modules/build-config-capability.adoc:7]: Should we provide some minimal description of what the Build API consists of? E.g. the same few lines as in https://github.com/openshift/api/blob/master/config/v1/types_cluster_version.go#L318:
```
Build API is responsible for watching the Build API objects and managing their lifecycle. The following resources are part of the API: builds, buildconfigs.
```

**Comment by ingvagabund** [comment]
LGTM

File: snippets/capabilities-table.adoc [hunk 2]
```diff
@@ -14,12 +14,14 @@ The following table describes the `baselineCapabilitySet` values.
...
+|Specify when you want the capabilities recommended in {product-title} 4.12 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.12 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage` and `CSISnapshot`.
 
 |`v4.13`
-|Specify when you want the capabilities recommended in {product-title} 4.13 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.12 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot` and `NodeTuning`. 
+|Specify when you want the capabilities recommended in {product-title} 4.13 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.13 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot` and `NodeTuning`.
+
+// TODO: Add `Build` and `DeploymentConfig` to the list for 4.14.
```
Comment (bergerhoffer) [inline snippets/capabilities-table.adoc:22]: @bscott-rh Here's the PR that adds to the cluster capabilities page, if you can PTAL to make sure it looks okay to you. And here's a reminder whenever you add the v4.14 link, to add `Build` and `DeploymentConfig` please.

**Comment by bergerhoffer** [comment]
@kasturinarra Can you PTAL at this PR for adding Build and DeploymentConfig capabilities?

Preview: https://63812--docspreview.netlify.app/openshift-enterprise/latest/installing/cluster-capabilities

File: snippets/capabilities-table.adoc (hunk 2 above)
Comment (bscott-rh) [reply snippets/capabilities-table.adoc:22]: The changes look good to me. Thank you

**Pull request merged**